subgraph/src/helpers.ts

```diff
@@ -34,3 +35,29 @@ export function getKeyExpirationTimestampFor(
     return lockContract.keyExpirationTimestampFor(ownerAddress)
   }
 }
+
+export function loadOrCreateUnlockDailyData(timestamp: number) {
+  const dayID = timestamp / 86400
+  let unlockDailyData = UnlockDailyData.load(dayID.toString())
+  const unlockDailyDataYesterday = UnlockDailyData.load((dayID - 1).toString())
+  if (unlockDailyData === null) {
```

Comment:
doesnt that condition mean that no action will be taken if yesterday data is not null?